source/hardware/controllers/reloop_mixage.rst

```diff
@@ -0,0 +1,115 @@
+6       :hwlabel:`GAIN`                            Preamplication of a track/deck                                                                                                                                              Same as non shift function
+7       Pitchfader                                 Adjust the pitch or playback speed of the current deck. The LED lights up in the neutral position                                                                           Same as non shift function
+8       :hwlabel:`RELOOP` / :hwlabel:`OUT`         Toggle relooping on/off. Button is lit when a loop is active, blinks slowly if there's an inactive loop set and thus reloop is available                                    Set loop out point
+9       :hwlabel:`FX SEL` / :hwlabel:`MASTER`      Select effect in effect rack N for adjustment. Button is lit if an effect slot is selected                                                                                  Toggle if deck N is sync master
+10      :hwlabel:`-DRY/WET+` / :hwlabel:`PAN`      Turn to control dry/wet for effect rack N; Turn whilst pushing down to select effect preset for effect rack N; Press shortly to toggle effect(s) in the effect rack on/off  Select quick effect preset for deck N. Click to toggle quick effect on/off
+11      :hwlabel:`AMOUNT` / :hwlabel:`FILTER`      Control effect meta knob when an effect slot is selected; Controls the effect rack meta knob when no effect slot is selected                                                Control quick effect super knob
+12      :hwlabel:`FX ON` / :hwlabel:`KEYLOCK`      Toggle effect rack on/off for deck N                                                                                                                                        Toggle keylock (keep pitch on speed change) on/off
+13      :hwlabel:`🔍`                              Toggle to use jog wheel to scroll through deck                                                                                                                              Same as non shift function
+14      :hwlabel:`💿`                              Toggle to use jog wheel to scratch deck                                                                                                                                     Same as non shift function
+15      Jog wheel                                  Controls various functions depending on the jog wheel mode                                                                                                                  Same as non shift function
+16      :hwlabel:`⯈⯇` / :hwlabel:`CUE 1`           Beat-sync deck as follower (hold to sync lock)                                                                                                                              If hot cue 1 is set, go to hot cue 1, else set hot cue 1 at current position
```

Comment:
I didn't check the mapping, but if that buttons is doing the same as Sync in the GUI it does a one-time beat sync (BPM), and sync lock on longpress. The new explicit Sync Leader control is not mapped, is it?

Reply:
Is the the sync leader functionality working in 2.4? It was b0rked in 2.3

Reply:
Yes. It's a different thing than the previous implementation.

Reply:
> Yes. It's a different thing than the previous implementation.

Sorry. I edited the comment in the meantime. Could you resolve if it's ok for you now?

Reply:
:+1: